src/components/utils/PromiseComponent/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-curly-brace-presence */
```

Comment:
I'd shutdown this lint check for the whole project (in `.eslintrc` file) since we're generally used to wrap string props between curly braces and also strings in children.

@davidebianchi what do you think about this matter?

Reply:
We usually use. the curly brace, so I would maintain this behaviour